ansible/update.yml

```diff
@@ -0,0 +1,38 @@
+---
+# Entry point for update action.
+
+- name: Setup AgnosticD Runtime
+  import_playbook: setup_runtime.yml
+
+- name: Start environment
+  import_playbook: >-
+    {{ lookup('first_found', {
+         'files': [ 'start.yml', 'lifecycle.yml' ],
+         'paths': ['configs/' + env_type, playbook_dir]
+       })
+    }}
+  vars:
+    ACTION: start
```

Comment:
Should we make the update action atomic?
(Managing chaining or dependency of state outside of the action itself).
I would tend to say yes.

In the context of babylon, anarchy would make sure the state is "started" before running the update.
In the context of developing or running from the laptop, you don't update a stopped environment.

Also, is there a use-case where we don't need to "start" the environment to update it? Like a shared cluster which would always be on?

Reply:
I would agree, but Ansible is slow and inefficient. If we schedule a start before update then this would likely add minutes to every user waiting for an environment and double the workload on Anarchy. So, while it would be more elegant to separate update from start, it is far more efficient to combine them.

Reply:
I see.

Then i would argue the place to add the start would be inside the config, wdyt?

Reply:
I think "started" is the final state we want to be after an "update"

But how we get there could depend on the config. Maybe the environment needs to be stopped in order to change the image of the instances, and then it's started.
So instead of:

1.  start the env
2. stop the env
3. change the AMI
4. start the AMI

it could just be 2,3,4. That's why i'm suggesting it's a config thing.

Reply:
Ah, that's a fair point. I'll update the PR.

Reply:
Updated